tests/python-tests/CWE-502/local/unsafe.py

```diff
@@ -7,7 +7,7 @@
 pickle.loads(i)
 
 # > Local Environment Variables
-e = pickle.dumps(os.environ.get('LOCAL_DATA'))
```

Comment:
Did you have to remove this one in order to make the test work? If so, this looks like a missing taint step?

Reply:
This was wrong. You need to load before dumping the content